src/buttons/button-icon.ts

```diff
@@ -18,4 +19,9 @@ export class NglButtonIcon {
     this.renderer.setElementClass(this.element.nativeElement, 'slds-button', true);
   }
 
-};
+  private normalize(type: string): string {
+    if (type == null) return '-border';
```

Comment:
Actually we need to leave out the empty string case, that's what the double equals is for.